### PR TITLE
Add styles for footer

### PR DESCRIPTION
**Pull request by kylefiedler**
screen shot 2014-11-13 at 2 35 35 pm

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -70,21 +70,24 @@
       </section>
 
       <footer>
-        <article>
+        <article class="thoughtbot">
           <h1>Site design &amp; development by</h1>
+          {% include thoughtbot.svg %}
```
Comment (creuter) [inline _layouts/default.html:75]: nice

File: _layouts/default.html [hunk 2]
```diff
@@ -105,8 +111,10 @@
                 <li><a href="">Incubators &amp; Accelerators</a></li>
               </ul>
             </li>
-            <li>
-              <a href="">Listserv</a>
+
+            <li class="group">
+              <h3><a href="">Listserv</a></h3>
```
Comment (creuter) [inline _layouts/default.html]: Break the `a` into its own line to be consistent with the `h3`s above.

File: _layouts/default.html [hunk 3]
```diff
@@ -105,8 +111,10 @@
...
+              <h3 class="twitter"><a href="https://twitter.com/startupleaders">Follow us on Twitter</a></h3>
```
Comment (creuter) [inline _layouts/default.html]: ditto

File: _sass/base/extends/_content-title.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+%content-title {
+  font-family: $base-font-family;
+  font-size: em(14);
+  letter-spacing: 1px;
+  margin-bottom: 1em;
+  text-transform: uppercase;
+}
+
+%content-sub-title {
```
Comment (creuter) [inline _sass/base/extends/_content-title.scss]: don't know that this should be hyphenated

File: _sass/layout/_footer.scss [hunk 5]
```diff
@@ -1,6 +1,80 @@
...
   margin-bottom: 3em;
-  padding: 4em;
+  padding: 4em 8em;
+}
+
+.thoughtbot {
+  @include clearfix;
+  padding-bottom: 4em;
+  margin-bottom: 4em;
+  border-bottom: 1px solid lighten($dark-brown, 10);
```
Comment (creuter) [inline _sass/layout/_footer.scss]: :abc:

File: _sass/layout/_footer.scss [hunk 6]
```diff
@@ -1,6 +1,80 @@
...
+      }
+    }
+  }
+
+  li {
+    margin-bottom: 0.5em;
+  }
+
+  .group {
+    @include span-columns(2 6);
```
Comment (creuter) [inline _sass/layout/_footer.scss:74]: is this the same as 2 of 6?

File: _sass/layout/_footer.scss [hunk 7]
```diff
@@ -1,6 +1,80 @@
...
+
+  h3 {
+    font-size: em(15);
+    font-weight: normal;
+    margin-bottom: 0.5em;
+
+    a {
+      color: white;
+
+      &:hover {
```
Comment (creuter) [inline _sass/layout/_footer.scss:63]: Is `focus` visible?

**Comment by creuter** [comment]
looking good

:ship:

File: _sass/layout/_footer.scss (hunk 6 above)
Comment (kylefiedler) [inline _sass/layout/_footer.scss:74]: Yes

**Pull request merged**